db/seeds.rb

```diff
@@ -2,4 +2,65 @@
+CrackedIneffectiveTrial.create(code: "H", description: "Defendant deceased", reason_type: "cracked")
+CrackedIneffectiveTrial.create(code: "I", description: "Prosecution end case: insufficient evidence", reason_type: "cracked")
+CrackedIneffectiveTrial.create(code: "J", description: "Prosecution end case: witness absent / withdrawn", reason_type: "cracked")
+CrackedIneffectiveTrial.create(code: "K", description: "Prosecution end case: public interest grounds", reason_type: "cracked")
+CrackedIneffectiveTrial.create(code: "L", description: "Prosecution end case: adjournment refused", reason_type: "cracked")
+CrackedIneffectiveTrial.create(code: "M1", description: "Prosecution offers no evidence not ready: served late notice of additional evidence on defence", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "M2", description: "Prosecution not ready: specify in comments", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "M3", description: "Prosecution failed to disclose unused material", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "N1", description: "Prosecution witness absent: police", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "N2", description: "Prosecution witness absent: professional / expert", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "N3", description: "Prosecution witness absent: other", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "O1", description: "Prosecution advocate engaged in another trial", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "O2", description: "Prosecution advocate failed to attend", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "P", description: "Prosecution increased time estimate - insufficient time for trial to start", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "Q1", description: "Defence not ready: disclosure problems", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "Q2", description: "Defence not ready: specify in comments (inc. no instructions)", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "Q3", description: "Defence asked for additional prosecution witness to attend", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "R", description: "Defence witness absent", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "S1", description: "Defendant absent - did not proceed in absence (judicial discretion)", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "S2", description: "Defendant ill or otherwise unfit to proceed", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "S3", description: "Defendant not produced by PECS", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "S4", description: "Defendant absent â€“ unable to proceed as Defendant not notified of place and time of hearing", reason_type: "ineffective")
```

Comment:
Could you replace the â€“ with a -?  The google sheet had â€“ but I think we're safe to switch to a - 
There are a couple in this data